h2o-algos/src/main/java/hex/tree/isoforextended/SubSampleTask.java

```diff
@@ -0,0 +1,37 @@
+package hex.tree.isoforextended;
+
+import water.MRTask;
+import water.fvec.Chunk;
+import water.fvec.NewChunk;
+import water.util.RandomUtils;
+
+import java.util.Random;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Task to create random sub-sample of given Frame with defined subSampleSize
+ * TODO avalenta make subsampling randomized and guarantee given size for small datasets
```

Comment:
do you plan on fixing this?

Reply:
Will be resolved on https://github.com/h2oai/h2o-3/pull/4907